bigquery_etl/view/__init__.py

```diff
@@ -192,7 +192,9 @@ def publish(self, target_project=None, dry_run=False):
             print(f"Skipping {self.path}")
             return True
 
-        if self.is_valid():
+        # avoid checking references since Jenkins might throw and exception:
+        # https://github.com/mozilla/bigquery-etl/issues/2246
+        if self._valid_view_naming():
```

Comment:
`is_valid()` calls `_valid_fully_qualified_references()` which uses pyjnius for checking references in views. Since Jenkins seems to run into problems there, we can skip that check and avoid using anything java.